### DISCUSSION TITLE
What determines an admin's ability to "modify" an update in the Activity Log?

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15103] [Archive]
I'm curious what determines which item updates can be modified and which are read-only.

https://user-images.githubusercontent.com/62306630/185000703-d63c9905-b07b-49ce-836e-e9b397edae6a.mov

**Comment by licitdev** [comment]
For non-system collections that do not begin with `directus_`.
https://github.com/directus/directus/blob/7c644934fe2cf9bd02e081c8c119d1809ff41ec6/app/src/modules/activity/routes/item.vue#L88-L89

**Reply by erondpowell** [reply]
Thanks Ian!
